sim/build.sbt

```diff
@@ -4,6 +4,30 @@ import Tests._
 val apiDirectory = settingKey[String]("The site directory into which the published scaladoc should placed.")
 apiDirectory := "latest"
 
+def filenames(tempDir: File, fs: Seq[File]): Seq[String] =
+for(f <- fs) yield {
+  sbtassembly.AssemblyUtils.sourceOfFileForMerge(tempDir, f) match {
+    case (path, base, subDirPath, false) => subDirPath
+    case (jar, base, subJarPath, true) => jar + ":" + subJarPath
+  }
+}
+
+val not_rc_merge_strategy = new sbtassembly.MergeStrategy {
```

Comment:
Camel casing + a brief comment explaining the intent of this code would be nice.

Reply:
Will do.

Reply:
Done.